src/containers/Balance/Balance.tsx

```diff
@@ -162,11 +162,7 @@ class Balance extends React.Component<Props, State> {
         <div className="actions">
           <button
             type="button"
-            disabled={
-              !amount ||
-              !Number.isFinite(Number(amount)) ||
-              (!toAddress && !toContact)
-            }
+            disabled={!amount || (!toAddress && !toContact)}
```

Comment:
Is there a way to check against a finite value for BN?

Or do we not care?

Reply:
We do not care